test/JDBC/input/BABEL-1199-vu-verify.sql

```diff
@@ -35,5 +35,11 @@ GO
 SELECT user_name(DATABASE_PRINCIPAL_ID('db_owner '));
 GO
 
-SELECT DATABASE_PRINCIPAL_ID('NULL')
+SELECT user_name(DATABASE_PRINCIPAL_ID());
+GO
+
+SELECT DATABASE_PRINCIPAL_ID('test_me')
+GO
+
+SELECT DATABASE_PRINCIPAL_ID(NULL)
```

Comment:
Please add dependency tests for the new user_id function as Surendra suggested. And I'll suggest you put all these tests in JDBC/input/functions directory, and name them after the functions, so that others can easily understand the test context from the title.